DuckDuckGoTests/DaxDialogFireEducationTests.swift

```diff
@@ -0,0 +1,248 @@
+//
+//  DaxDialogTests.swift
+//  UnitTests
+//
+//  Copyright © 2020 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import XCTest
+@testable import DuckDuckGo
+@testable import Core
+
+class DaxDialogFireEducationTests: XCTestCase {
```

Comment:
I decided it would be easier to clean up once the experiment is done if I separated the tests for when the fire education experiment is on into a separate class, rather than making them part of DaxDialogTests